include/mbedtls/ssl.h

```diff
@@ -2373,18 +2375,49 @@ void mbedtls_ssl_conf_session_cache( mbedtls_ssl_config *conf,
 
 #if defined(MBEDTLS_SSL_CLI_C)
 /**
- * \brief          Request resumption of session (client-side only)
- *                 Session data is copied from presented session structure.
- *
- * \param ssl      SSL context
- * \param session  session context
+ * \brief          Load a session for session resumption.
+ *
+ *                 Sessions loaded through this call will be considered
+ *                 for session resumption in the next handshake.
+ *
+ * \note           Even if this call succeeds, it is not guaranteed that
+ *                 the next handshake will indeed be shortened through the
+ *                 use of session resumption: The server is always free
+ *                 to reject any attempt for resumption and fall back to
+ *                 a full handshake.
+ *
+ * \note           The mechanism of session resumption is opaque to this
```

Comment:
Bit of a nit, but IIUC the word 'transparent' was used instead of 'opaque' in the mailing list email, to mean the same thing. Could this be confusing? (Also in ln 3721)

Reply:
Hm good point - I _think_ "opaque" is the right word here, though, no?

Reply:
I'm not sure what you mean to say here. If you mean “this function can handle all session resumption mechanisms supported by Mbed TLS”, then, well, you have my alternative wording suggestion.

Reply:
Hmm
On reflection, I agree. Language is strange, I suppose.